coalib/parsing/filters/FilterBase.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+
+
+def overrides(interface_class):
+    def overrider(method):
+        assert(method.__name__ in dir(interface_class))
+        return method
+    return overrider
```

Comment:
Or may be this `overrides` decorator can be kept in some utility class/function so that whole codebase can use it easily.